### DISCUSSION TITLE
tugas pak komang

### DISCUSSION THREAD

**Discussion by ipfa24** [Rafli-Dewanto/project-android-pbo#103] [Q&A] [answered]
permisi fli, gua imron temen wardi ini tugas pak komang suruh ngapain dah?

**Comment by Rafli-Dewanto** [comment, marked as answer]
tugas yang repo ini? klo kelas gua disuruh nyoba-nyoba doang si, terus dia bilang suruh redesign tiap halaman karena design-nya terlalu kaku

**Reply by ipfa24** [reply]
kalau redesign berarti xml aja kan ya?
atau kita tambah fitur gak apa apa?

**Reply by Rafli-Dewanto** [reply]
tambah fitur juga boleh katanya

**Reply by ipfa24** [reply]
okelah thanks info makasih fli👌🏻

**Reply by Rafli-Dewanto** [reply]
👍
